### PR TITLE
DOP-2425: Update padding and margins for TOC

### PR DESCRIPTION
**Pull request by rayangler**
### Stories/Links:

DOP-2425

### Staging Links:

[Docs Staging](https://docs-mongodbcom-staging.corp.mongodb.com/master/docs/raymundrodriguez/DOP-2425/)
[Landing Staging](https://docs-mongodbcom-staging.corp.mongodb.com/upstream-nav/landing/raymundrodriguez/DOP-2425/)
[Realm Staging](https://docs-mongodbcom-staging.corp.mongodb.com/master/realm/raymundrodriguez/DOP-2425/)

### Notes:
* Updated padding and margins to save vertical space and to make nested TOC items more visibly distinct from parents.

### CODE REVIEW

**Review by allisonmui9 — APPROVED** [review]
LGTM thank you raymund!

File: src/components/VersionDropdown.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ const zip = (a, b) => {
 };
 
 const StyledSelect = styled(Select)`
-  margin: ${theme.size.default} ${theme.size.default} ${theme.size.medium} ${theme.size.default};
+  margin: 8px ${theme.size.medium} 8px ${theme.size.medium};
```
Comment (mango-db) [inline src/components/VersionDropdown.js:21]: nit: `theme.size.small`

**Review by mango-db — APPROVED** [review]
🍓 LGTM 🍓 

I'd be curious to see how this looks on our most deeply nested pages (I was only able to find 3-deep nesting), but I don't think that should hold this up

**Comment by rayangler** [comment]
@allisonmui9 @mango-db I updated the PR and staging links with a fix for the visual bugs I was running into on [Landing](https://docs-mongodbcom-staging.corp.mongodb.com/upstream-nav/landing/raymundrodriguez/DOP-2425/). Please re-review whenever you're available. Thank you!

**Review by allisonmui9 — APPROVED** [review]
lgtm! thank you! :)

File: src/components/Sidenav/Sidenav.js [hunk 2]
```diff
@@ -216,20 +206,26 @@ const Sidenav = ({ page, pageTitle, publishedBranches, siteTitle, slug, toctree
...
-              </SiteTitle>
+              </SideNavItem>
             )}
             {publishedBranches && <VersionDropdown slug={slug} publishedBranches={publishedBranches} />}
             {!ia && <Toctree handleClick={() => hideMobileSidenav()} slug={slug} toctree={toctree} />}
 
             {isDocsLanding && (
               <>
                 <Spaceholder />
+                {/* Represents the generic links at the bottom of the side nav (e.g. "Contact Support") */}
```
Comment (mango-db) [inline src/components/Sidenav/Sidenav.js:219]: TY for preserving comments 🙏

**Review by mango-db — APPROVED** [review]
Looks guava to me 🥭 

- [x] Functions as expected on develop mode on landing
- [x] Deeply-nested pages such as [this one](https://docs-mongodbcom-staging.corp.mongodb.com/master/docs/raymundrodriguez/DOP-2425/installation/) LGTM

**Pull request merged**